etc/ibm-cloud-sdk-core.api.md

```diff
@@ -13,10 +13,10 @@ import { OutgoingHttpHeaders } from 'http';
 import { Stream } from 'stream';
 
 // @public
-export function atLeastOne(a: any, b: any): boolean;
+export function atLeastOne(...args: any): boolean;

```

Comment:
Here we see the API change but note that they are compatible: `...args: any` can very well equal `a: any, b: any`.